projects/common/src/utilities/lang/lang-utils.ts

```diff
@@ -31,3 +31,6 @@ const ignoreFunctions = (first: unknown, second: unknown) => {
     return first.toString() === second.toString();
   }
 };
+
+export const isEmptyString = (str: string | undefined | null): boolean =>
```

Comment:
Do we not want to use `isEmpty` ? I'm generally for writing our own utils, but we've used lodash directly pretty extensively

Reply:
Ah, didn't realize that method picked up empty quotes. Just tested and it does. I'll remove that method. The other I added after you reviewed is more helpful anyway. I'll keep that one.